appserver/packager/rest-monitoring/pom.xml

```diff
@@ -53,6 +53,7 @@
 
     <properties>
         <temp.dir>${project.build.directory}/dependency</temp.dir>
+        <rest.monitoring.version>1.0.0-Beta</rest.monitoring.version>
```

Comment:
What was wrong with this being at the top-level? Placing it under packager is particularly odd.

Reply:
An issue with the maven-release-plugin that results in failure when using inherited properties as version number definitions which aren't project.version - very old problem but doesn't seen to have been fixed (see https://issues.apache.org/jira/browse/MRELEASE-782)